avocado/plugins/replay.py

```diff
@@ -234,7 +234,9 @@ def run(self, config):
                               'ignore_missing_references',
                               'execution_order',
                               'loaders',
-                              'external_runner']:
+                              'external_runner',
+                              'external_runner_testdir',
```

Comment:
It's a small issue, but this line should be in the commit "migrate --external-runner-testdir to use the new settings API" commit (or if you want merge the commits, it's also fine).